ghost/domain-events/lib/DomainEvents.js

```diff
@@ -28,7 +28,7 @@ class DomainEvents {
      * @returns {void}
      */
     static subscribe(Event, handler) {
-        DomainEvents.ee.on(Event.name, handler);
```

Comment:
What's the reasoning for switching this?

Previously we were able to pass these methods around as functions individually, but with this change we can't do that without creating a new function with the `this` context bound.

I don't think we're using that feature anywhere, but curious why we'd change this

Reply:
Oh, I didn't know we passed it around as functions. I can't find a reference to it. I thought using 'this' was cleaner than repeating the constructor name. Also not sure if it is a good pattern to not have 'this' bound, because that requires you to know the implementation of the method and makes it more dangerous to change the implementation later. I'll revert this change for now, to be sure we don't break anything